dcos/dcos-ui
# DCOS-46535: Translate Jobs Modal

Mark and translate to Chinese jobs modal tabs.

Closes https://jira.mesosphere.com/browse/DCOS-46535

## Testing
1. Switch to Chinese.
2. Open the Jobs tab (the third one).
3. Click the purple button to create a new job.
4. Observe that it is (partially for now) translated

## Trade-offs
Could not find a better way to translate it than to use https://github.com/dcos/dcos-ui/compare/gt/chore/DCOS-46535-translate-jobs-modal?expand=1#diff-bbf4b4167e8dc33b2559f2b026066aecR5 .

## Dependencies
None.

## Screenshots

### Before
![screenshot from 2019-01-02 14-08-48](https://user-images.githubusercontent.com/40791275/50591369-f2ed4f80-0e97-11e9-89e7-a16552d41dc1.png)

### After
![2019-01-02 22-06-41](https://user-images.githubusercontent.com/40791275/50610184-b3942280-0eda-11e9-85c2-9aeaf799f96c.png)
